dora/core/common/src/main/java/alluxio/UfsUrlUtils.java

```diff
@@ -0,0 +1,143 @@
+package alluxio;
+
+import alluxio.grpc.MountPointInfo;
+import alluxio.grpc.UfsUrlMessage;
+import alluxio.uri.Authority;
+import alluxio.uri.URI;
+
+import alluxio.uri.UfsUrl;
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Preconditions;
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.logging.log4j.util.Strings;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ *
+ * TODO: implement the equivalence of URI
+ * TODO: implement the equivalence of PathUtils
+ * TODO: implement the equivalence of AlluixoURI
+ * TODO: move/rename the UTs of above, to make sure we have guaranteed the same functionalities
```

Comment:
remove the TODOs if they are resolved. For example, check the methods in `PathUtils` and see if those functionalities have been covered.